instance/static/html/instance/details.html

```diff
@@ -28,6 +28,22 @@ <h3 ng-if="instance">
   </div>
 </div>
 
+<div class="panel" ng-controller="Details">
+  <div class="row">
+    <div class="large-10 columns">
+      <h3>Notes <a ng-click="is_editing_notes=true"><i class="fa fa-edit"></i></a></h3>
+
+      <div ng-if="!is_editing_notes">{{ instance.notes }}</div>
```

Comment:
Can we support markdown here? Maybe via sending it through [marked.js](https://marked.js.org) here in the frontend? We should also consider sanitizing, otherwise it leaves it open to XSS.

Reply:
Definetly! I'll take a look at it